functests/1_performance/hugepages.go

```diff
@@ -44,7 +44,9 @@ var _ = Describe("[performance]Hugepages", func() {
 
 		profile, err = profiles.GetByNodeLabels(testutils.NodeSelectorLabels)
 		Expect(err).ToNot(HaveOccurred())
-		Expect(profile.Spec.HugePages).ToNot(BeNil())
+		if profile.Spec.HugePages == nil || profile.Spec.HugePages.Pages == nil {
```

Comment:
can you please change `profile.Spec.HugePages.Pages == nil` to `len(profile.Spec.HugePages.Pages == 0)`, I just unsure when under the API the list value is nil and when it not nil but does not have any elements, so the len check will cover both cases

Reply:
hm, I was afraid if we can catch "nil pointer exceptions" in that case..
verified locally - works fine, thanks. Updated.